op-node/withdrawals/utils.go

```diff
@@ -264,23 +267,17 @@ func ParseWithdrawalInitiated(receipt *types.Receipt) (*bindings.L2ToL1MessagePa
 	if err != nil {
 		return nil, err
 	}
-	abi, err := bindings.L2ToL1MessagePasserMetaData.GetAbi()
-	if err != nil {
-		return nil, err
-	}
 
 	for _, log := range receipt.Logs {
```

Comment:
this still only parses the first withdrawal that is done (& would second multiple withdrawals if that is possible to do in a single transaction). Probably fine for now, but good to keep track of.

Reply:
Added in docstring, and tracking as ENG-2852.